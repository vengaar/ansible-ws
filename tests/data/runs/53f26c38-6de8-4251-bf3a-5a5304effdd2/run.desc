{"playbook": "/home/vengaar/wapi/test/playbooks/test_nowapi.yml", "cmdline": "ansible-playbook /home/vengaar/wapi/test/playbooks/test_nowapi.yml "}