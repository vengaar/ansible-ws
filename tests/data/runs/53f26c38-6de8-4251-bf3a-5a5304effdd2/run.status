{"pid": 19675, "runid": "53f26c38-6de8-4251-bf3a-5a5304effdd2", "status": "finished", "state": "succeeded", "begin": 1551460751.3591459, "end": 1551460753.9493072, "return_code": 0, "description": {"playbook": "/home/vengaar/wapi/test/playbooks/test_nowapi.yml", "cmdline": "ansible-playbook /home/vengaar/wapi/test/playbooks/test_nowapi.yml "}}