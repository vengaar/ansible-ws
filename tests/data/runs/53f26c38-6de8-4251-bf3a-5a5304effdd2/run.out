
PLAY [localhost] ***************************************************************

TASK [Display inventory_hostname] **********************************************
[0;32mok: [localhost] => {[0m
[0;32m    "inventory_hostname": "localhost"[0m
[0;32m}[0m

TASK [Display param email] *****************************************************
[0;32mok: [localhost] => {[0m
[0;32m    "email": "VARIABLE IS NOT DEFINED!"[0m
[0;32m}[0m

TASK [Display param app1_server] ***********************************************
[0;32mok: [localhost] => {[0m
[0;32m    "app1_server": "VARIABLE IS NOT DEFINED!"[0m
[0;32m}[0m

TASK [Display param prod_servers] **********************************************
[0;32mok: [localhost] => {[0m
[0;32m    "prod_servers": "VARIABLE IS NOT DEFINED!"[0m
[0;32m}[0m

TASK [Display param color] *****************************************************
[0;32mok: [localhost] => {[0m
[0;32m    "color": "VARIABLE IS NOT DEFINED!"[0m
[0;32m}[0m

TASK [Display param agree] *****************************************************
[0;32mok: [localhost] => {[0m
[0;32m    "agree": "VARIABLE IS NOT DEFINED!"[0m
[0;32m}[0m

TASK [Task with tag] ***********************************************************
[0;32mok: [localhost] => {[0m
[0;32m    "msg": "A tagged task"[0m
[0;32m}[0m

TASK [Task without tag] ********************************************************
[0;32mok: [localhost] => {[0m
[0;32m    "msg": "A task without tags"[0m
[0;32m}[0m

PLAY RECAP *********************************************************************
[0;32mlocalhost[0m                  : [0;32mok=8   [0m changed=0    unreachable=0    failed=0   

